diff --git a/crates/bevy_diagnostic/src/frame_time_diagnostics_plugin.rs b/crates/bevy_diagnostic/src/frame_time_diagnostics_plugin.rs
index 99a727061..eb1049ccb 100644
--- a/crates/bevy_diagnostic/src/frame_time_diagnostics_plugin.rs
+++ b/crates/bevy_diagnostic/src/frame_time_diagnostics_plugin.rs
@@ -14,10 +14,14 @@ pub struct FrameTimeDiagnosticsPlugin;
 
 impl Plugin for FrameTimeDiagnosticsPlugin {
     fn build(&self, app: &mut App) {
-        app.register_diagnostic(Diagnostic::new(Self::FRAME_TIME).with_suffix("ms"))
-            .register_diagnostic(Diagnostic::new(Self::FPS))
-            .register_diagnostic(Diagnostic::new(Self::FRAME_COUNT).with_smoothing_factor(0.0))
-            .add_systems(Update, Self::diagnostic_system);
+        app.register_diagnostic(
+            Diagnostic::new(Self::FRAME_TIME)
+                .with_suffix("ms")
+                .with_max_history_length(9999),
+        )
+        .register_diagnostic(Diagnostic::new(Self::FPS).with_max_history_length(9999))
+        .register_diagnostic(Diagnostic::new(Self::FRAME_COUNT).with_smoothing_factor(0.0))
+        .add_systems(Update, Self::diagnostic_system);
     }
 }
 
